Practice2/src/partials/features-section.html

```diff
@@ -0,0 +1,47 @@
+<section class="page-features">
+  <h1 class="features-heading">Features</h1>
+  <h4 class="features-heading-sub">We're talking it to the next level</h4>
+  <div class="features-social">
+    <div class="features-description">
+      <div class="media-features media-features-warning">
+        <img src="./assets/images/features-sub-primary.jpg" alt="Icon" class="media-features-thumbnail">
+        <div>
+          <h4 class="media-text media-title-features">
+            Reponsive Layout
```

Comment:
Please help to double-check the typo here.

Reply:
What do you mean? What is " typo " ?

Reply:
What do you mean? What is " typo " ?

Reply:
@duylong13 Vuong once explained this word before. Trying to search the word in google first, there are so many articles over there for this.